plutus-core/untyped-plutus-core/Language/UntypedPlutusCore/Core/Instance/Flat.hs

```diff
@@ -0,0 +1,66 @@
+{-# LANGUAGE LambdaCase           #-}
+{-# LANGUAGE TypeOperators        #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Language.UntypedPlutusCore.Core.Instance.Flat where
```

Comment:
Comments. Also probably reference the notes from the other file.

Reply:
I have added all the comments from the typed instances and added a pointer to the typed instances (since we are using some of the instances defined there in here).